<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCartButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Cart</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Class</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Special Attribute</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Bundle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Savings</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
